assembl/static2/js/app/components/form/utils.js

```diff
@@ -14,7 +15,22 @@ export function i18nValueIsEmpty(v: I18nValue): boolean {
   );
 }
 
-export function convertEntries(_entries: LangstringEntries): I18nValue {
+export function richTextI18nValueIsEmpty(v: I18nRichTextValue): boolean {
+  return (
+    !v ||
+    Object.keys(v)
```

Comment:
Unfortunately, we need to do this because of this issue in flow : https://github.com/facebook/flow/issues/2221
We already done this multiple times in the code. We could add a TODO comment or put this in a global function so it will be simpler to change when flow is fixed